### ISSUE TITLE
Immediate redirect to Magento root

### ISSUE THREAD

**Issue by arnekolja** [witrin/TypoGento#33]
Hey there,

I am struggling with TYPO3-side configuration and cannot seem to get the generic routes (tried quite a few setups, now sticking with https://gist.github.com/witrin/3220650) running.

There were mainly two errors. One saying that the dispatch route is not configured properly and now with the gist mentioned above there is no error shown, but I get redirected to the shop whenever opening a page that contains the TypoGento plugin.

Do you have any idea on how to resolve this?
- TYPO3 6.1.1
- TypoGento latest revision from this repo
- config.tx_typogento.header.enable = 0
- Plugin configured to show a specific category (tried single products, same result)
- All reports are green
- Magento backend integration in TYPO3 works fine

**Comment by witrin** [comment]
I've not tested ever the old plugin with TYPO3 6. To check what your routing setup is producing place `echo 'Dispatch URL: ' . $this->_url;` right after [this](https://github.com/witrin/TypoGento/blob/develop/src/TYPO3/typo3conf/ext/typogento/lib/class.tx_typogento_interface.php#L25). I vaguely remember this kind of a problem but not the solution. A debugger might be helpful.

In addition, I am very sad that I can not publish the latest version of TypoGento that supports beside Extbase and namespaces also a one-click setup, since I still lack the synchronization and authentication of accounts, and I can not find the time to finish this off nether anyone would like to help. Especially a useful concept for the synchronization doesn't exist and the master-master replication is too complex to implement.

**Comment by arnekolja** [comment]
By "the old plugin" you mean before your non-published extbase revamp?

I will try debugging it this way and come back here.

**Comment by witrin** [comment]
Yes, respectively the latest revision from the development branch.

**Comment by arnekolja** [comment]
The result of your tip above is <code>Dispatch URL: /catalog/category/view/id/37/?</code>. Seems correct to me. Nevertheless it is redirecting me to <code>http://www.myshop.com/</code>.

**Comment by witrin** [comment]
The problem was caused by the configuration of Magento and must be considered in the Documentation. In the backend of Magento  `System > Configuration > Web > Auto-redirect to Base URL` must be set to `No`.
